src/Plugin/Chorus/CMakeLists.txt

```diff
@@ -1,40 +1,21 @@
+dpf_add_plugin(ZynChorus
+    TARGETS clap lv2 vst2 vst3
+    FILES_DSP
+        Chorus.cpp
+)
 
-include_directories(${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_SOURCE_DIR}/DPF/distrho .)
+include_directories(.)
```

Comment:
this should probably use [target_include_directories](https://cmake.org/cmake/help/latest/command/target_include_directories.html)

Reply:
Did you mean `target_include_directories(ZynChorus ".")`?

Reply:
yes exactly